### PR TITLE
Update Ukrainian locale

### PR DESCRIPTION
**Pull request by gatezh**
- Add new translations
- Fix translation mistakes / inconsistencies

### CODE REVIEW

**Comment by paskal** [comment]
@umputun sorry, I don't speak Ukrainian and can't review it.

File: frontend/app/locales/ua.json [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
   "comment.blocked-user": "Заблокованний",
   "comment.blocking-period": "Період блокування",
   "comment.cancel": "Відмінити",
-  "comment.controversy": "Спірність: {value}",
+  "comment.controversy": "Суперечливість: {value}",
   "comment.copied": "Скопійовано!",
   "comment.copy": "Скопіювати",
   "comment.delete": "Видалити",
   "comment.delete-message": "Видалити коментар?",
   "comment.deleted-comment": "Коментар був видалений",
```
Comment (vosi) [inline frontend/app/locales/ua.json:37]: "Коментар було видалено"

File: frontend/app/locales/ua.json [hunk 2]
```diff
@@ -79,35 +79,35 @@
...
   "errors.12": "Не вдалося проголосувати. Спробуйте ще раз пізніше.",
   "errors.13": "Ви не можете голосувати за свій коментар.",
   "errors.14": "Ви вже голосували за цей коментар.",
   "errors.15": "Занадто багато голосів за коментар .",
   "errors.16": "Коментар досяг мінімальної оцінки.",
-  "errors.17": "Дія відклонена. Спробуйте ще раз пізніше.",
-  "errors.18": "Запрашиваемый файл не найден.",
-  "errors.19": "Comment contains restricted words.",
+  "errors.17": "Дію відхилено. Спробуйте ще раз пізніше.",
+  "errors.18": "Запитуваний файл не знайдено.",
```
Comment (vosi) [inline frontend/app/locales/ua.json:93]: "Бажаний файл не знайдено"

File: frontend/app/locales/ua.json [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
-  "auth.open-profile": "Open My Profile",
+  "auth.oauth-button": "Увійти через {provider}",
+  "auth.oauth-source": "Використати соцільну мережу",
+  "auth.open-profile": "Відкрити мій профіль",
   "auth.or": "або",
   "auth.signin": "Увійти",
   "auth.signout": "Вийти",
   "auth.submit": "Відправити",
-  "auth.symbols-restriction": "Ім’я користувача повинно починатися з літери і містити тільки латинські букви,цифри,знаки підкреслення і прогалини",
+  "auth.symbols-restriction": "Ім’я користувача повинно починатися з літери і містити тільки символи латинського алфавіту, цифри, знаки підкреслення або пробіли",
```
Comment (vosi) [inline frontend/app/locales/ua.json:12]: "...літери та містити..."

File: frontend/app/locales/ua.json [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
   "blockingDuration.day": "На день",
   "blockingDuration.month": "На місяць",
   "blockingDuration.permanently": "Назавжди",
-  "blockingDuration.week": "На неділю",
+  "blockingDuration.week": "На тиждень",
   "comment.block": "Блокувати",
   "comment.block-user": "Заблокувати користувача {userName} {duration}?",
   "comment.blocked-user": "Заблокованний",
   "comment.blocking-period": "Період блокування",
   "comment.cancel": "Відмінити",
```
Comment (zenitros) [inline frontend/app/locales/ua.json:31]:
```suggestion
  "comment.cancel": "Скасувати",
```

File: frontend/app/locales/ua.json [hunk 5]
```diff
@@ -79,35 +79,35 @@
...
   "errors.14": "Ви вже голосували за цей коментар.",
   "errors.15": "Занадто багато голосів за коментар .",
   "errors.16": "Коментар досяг мінімальної оцінки.",
-  "errors.17": "Дія відклонена. Спробуйте ще раз пізніше.",
-  "errors.18": "Запрашиваемый файл не найден.",
-  "errors.19": "Comment contains restricted words.",
+  "errors.17": "Дію відхилено. Спробуйте ще раз пізніше.",
+  "errors.18": "Запитуваний файл не знайдено.",
+  "errors.19": "Коментар містить заборонені слова.",
   "errors.2": "Не вдалося обробити відповідь від сервера.",
```
Comment (zenitros) [inline frontend/app/locales/ua.json:95]:
```suggestion
  "errors.2": "Не вдалося опрацювати відповідь від сервера.",
```

File: frontend/app/locales/ua.json [hunk 6]
```diff
@@ -79,35 +79,35 @@
...
+  "empty-state": "Поки немає коментарів",
   "errors.0": "Щось пішло не так,спробуйте ще раз пізніше.",
-  "errors.1": "Коментар не знайдений. Перезавантажте сторінку і спробуйте ще раз.",
+  "errors.1": "Коментар не знайдено. Перезавантажте сторінку і спробуйте ще раз.",
   "errors.10": "Час редагування коментаря минув.",
   "errors.11": "На ваш коментар вже відповіли, редагування недоступне.",
   "errors.12": "Не вдалося проголосувати. Спробуйте ще раз пізніше.",
   "errors.13": "Ви не можете голосувати за свій коментар.",
   "errors.14": "Ви вже голосували за цей коментар.",
   "errors.15": "Занадто багато голосів за коментар .",
```
Comment (zenitros) [inline frontend/app/locales/ua.json:90]:
```suggestion
  "errors.15": "Забагато голосів за коментар .",
```

File: frontend/app/locales/ua.json [hunk 7]
```diff
@@ -1,46 +1,46 @@
 {
   "auth.back": "Назад",
-  "auth.email-address": "Email адрес",
+  "auth.email-address": "Email адреса",
   "auth.loading": "Завантаження...",
-  "auth.oauth-button": "Sign In with {provider}",
-  "auth.oauth-source": "Use Social Network",
-  "auth.open-profile": "Open My Profile",
+  "auth.oauth-button": "Увійти через {provider}",
```
Comment (zenitros) [inline frontend/app/locales/ua.json:5]:
```suggestion
  "auth.oauth-button": "Увійти за допомогою {provider}",
```

**Comment by gatezh** [comment]
Applied all of the feedback. Ready for another review

**Review by akellbl4 — APPROVED** [review]
Changes to the code are applied correctly ;)

**Pull request merged**